docker/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+# docker-compose to make it easier to spin up integration tests.
```

Comment:
@dzmitry-kankalovich  I added a docker compose file under /docker we might move it at some point, but thinking about making it easier for developers to spin up services that integration tests depend on

Reply:
yeah that would be super useful!